test_runner/regress/test_pageserver_layer_rolling.py

```diff
@@ -247,9 +247,10 @@ def test_total_size_limit(neon_env_builder: NeonEnvBuilder):
 
     compaction_period_s = 10
 
+    checkpoint_distance = 1024**3
```

Comment:
For my curiosity, what broke the test?

Reply:
The `does_timeline_should_roll_prevent_failure`

Reply:
ooh yeah. it was 1024 ^ 4. I read it as 1024 * 4 somehow.